lib/compress/zstd_compress.c

```diff
@@ -1039,7 +1039,7 @@ size_t ZSTD_CCtx_loadDictionary_advanced(
     return 0;
 }
 
-ZSTDLIB_API size_t ZSTD_CCtx_loadDictionary_byReference(
+ZSTDLIB_STATIC_API size_t ZSTD_CCtx_loadDictionary_byReference(
```

Comment:
These prefixes (`ZSTDLIB_API`, `ZSTDLIB_STATIC_API`) shouldn't be needed at this stage in `zstd_compress.c`.
They should remain confined within `zstd.h`.
So just remove them.

Reply:
Done!